src/robusta/core/playbooks/internal/discovery_events.py

```diff
@@ -45,7 +45,7 @@ def event_history(event: ExecutionBaseEvent):
         )
         if events_table:
             finding.add_enrichment([events_table])
-            event.add_finding(finding)
+            event.add_finding(finding, True)
```

Comment:
Why is this supression necessary here?

Reply:
This is the get events history scenario.
We're creating multiple findings in this playbook, we're reading all warning events, and create a new finding for each.
That's ok in this scenario.
I've seen this warning for all new customers, on the first installation. this suppression hides it